ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -522,6 +522,9 @@ tests:
   cron: '@daily'
   steps:
     cluster_profile: packet-assisted
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
```

Comment:
I'd expect "explicit" meaning "ocp 4.10". Latest may change without a notice tbh.

Lets create a new "stable-4-10" tag and use it here?

Reply:
(please note that for specific spoke versions we have specific tests, e.g. `e2e-ai-operator-ztp-ipv4-sno-ocp-410` that installs 4.10 spoke or `e2e-ai-operator-ztp-ipv4v6-3masters-periodic-ocp-49` that installs 4.9 spoke)

Reply:
Oh, "explicitly" in this context means "pass payload pullspec" instead of relying on the defaults? That makes sense

Reply:
Right right right, they were not defined at all so we were running with whatever `dependency` specified as default